src/components/ButtonGroup/GroupButton/index.tsx

```diff
@@ -35,7 +35,7 @@ export interface GroupButtonProps {
    * `wrapper` must render the children prop passed into it
    * to render the GroupButton.
    */
-  Wrapper?: () => JSX.Element;
+  Wrapper?: (props: { children: React.ReactNode }) => JSX.Element;
```

Comment:
[should] Wrapper and WrapperComponent should probably use a shared type.

Reply:
can you elaborate on this? which one is WrapperComponent?

Reply:
Same file, up higher.

Reply:
fixed